after/ftplugin/php.vim

```diff
@@ -20,6 +20,14 @@ if !exists('g:hack#hh_client')
   let g:hack#hh_client = 'hh_client'
 endif
 
+if !exists('g:hack#hh_format') && executable('hh_format')
+  let g:hack#hh_format = 'hh_format'
+endif
+
+if !exists('g:hack#root')
+  let g:hack#root = '~/www'
+endif
```

Comment:
if not provided, the hh_format will complain about not having a default folder set - we can use the current folder, np